source/api/auth/0.9/index.md

```diff
@@ -597,11 +600,10 @@ Please note that the server implementation involves providing `302` status respo
 
 ### A. Implementation Notes
 
- * Care is required to implement this specification in a way that does not expose credentials thus compromising the security of the resources intended to be protected, or other resources within the same security domain.
- * Services using authentication should use HTTPS, and thus clients should also be run from pages served via HTTPS.
- * Implementations must not reuse the access cookie value as the access token value, as it could be copied across domains if the access token were to be obtained from a malicious client.
- * _further notes here_
-
+  * Services using authentication _SHOULD_ use HTTPS ([HTTP over TLS][rfc-2818]) for all interactions, and thus clients should also be run from pages served via HTTPS.
```

Comment:
We use RFC words in the implementation notes in the Image API spec (http://iiif.io/api/image/2.1/#a-implementation-notes) and in the Prezi API spec (http://iiif.io/api/presentation/2.1/#d-implementation-notes). To me it seems inconsistent to switch back and forth.